### PR TITLE
Code Cleanup 2020

### LINKED ISSUE #1412: Code Cleanup Tracker 2020
The codebase has some minor issues which keep bugging me in IntelliJ/PRs.

- [x] Add `final` to all fields which should have it
- [x] Add more checkstyle rules to align the formatting of the codebase together.
- [x] Replace all `\r\n` with `\n\`, except in `gradlew.bat` :P

### PR DESCRIPTION
**Pull request by octylFractal**
Closes #1412, obviously. Re-uses its tracker as well. Will be un-drafted when all tasks have been completed.

### CODE REVIEW

**Comment by octylFractal** [comment]
CRLF -> LF conversion:
it_cost_git_blame

File: worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/BukkitCommandSender.java [hunk 1]
```diff
@@ -76,21 +74,21 @@ public void printRaw(String msg) {
     @Override
     public void print(String msg) {
         for (String part : msg.split("\n")) {
-            sender.sendMessage("\u00A7d" + part);
+            sender.sendMessage("§d" + part);
```
Comment (me4502) [inline worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/BukkitCommandSender.java:77]: Is it fine to have these in the code now? It never used to be years back

File: worldedit-core/src/main/java/com/sk89q/worldedit/world/block/BaseBlock.java [hunk 2]
```diff
@@ -188,11 +189,7 @@ public int hashCode() {
 
     @Override
     public String toString() {
-//        if (getNbtData() != null) { // TODO Maybe make some JSON serialiser to make this not awful.
-//            return blockState.getAsString() + " {" + String.valueOf(getNbtData()) + "}";
-//        } else {
-            return blockState.getAsString();
-//        }
+        return blockState.getAsString() + (hasNbtData() ? "{hasNbt}" : "");
```
Comment (me4502) [inline worldedit-core/src/main/java/com/sk89q/worldedit/world/block/BaseBlock.java:192]: I feel the TODO is still worthwhile to have here. And pretty sure adventure (kyori text next version) includes an NBT to string serialiser anyway

File: worldedit-core/src/main/java/com/sk89q/worldedit/regions/selector/ExtendingCuboidRegionSelector.java [hunk 3]
```diff
@@ -122,9 +122,9 @@ public boolean selectSecondary(BlockVector3 position, SelectorLimits limits) {
         region.setPos1(position1);
         region.setPos2(position2);
 
-        assert(region.contains(o1));
-        assert(region.contains(o2));
-        assert(region.contains(position));
+        assert (region.contains(o1));
```
Comment (me4502) [inline worldedit-core/src/main/java/com/sk89q/worldedit/regions/selector/ExtendingCuboidRegionSelector.java:125]: Is this a method call? Or do we actually have assert statements in our code here

Comment (octylFractal) [reply worldedit-core/src/main/java/com/sk89q/worldedit/regions/selector/ExtendingCuboidRegionSelector.java:125]: These are assert statements. I suppose we could remove the `()` for clarity.

File: worldedit-core/src/main/java/com/sk89q/worldedit/world/block/BaseBlock.java (hunk 2 above)
Comment (octylFractal) [reply worldedit-core/src/main/java/com/sk89q/worldedit/world/block/BaseBlock.java:192]: Okay, I'll restore it.

File: worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/BukkitCommandSender.java (hunk 1 above)
Comment (octylFractal) [reply worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/BukkitCommandSender.java:77]: I have made it safe now, enforcing UTF-8 source-reading in all cases.

File: worldedit-core/src/main/java/com/sk89q/worldedit/regions/selector/ExtendingCuboidRegionSelector.java (hunk 3 above)
Comment (octylFractal) [reply worldedit-core/src/main/java/com/sk89q/worldedit/regions/selector/ExtendingCuboidRegionSelector.java:125]: I have done so.

**Pull request merged**